asset/css/search-editor.less

```diff
@@ -57,21 +57,21 @@
 
   .search-error {
     input:invalid {
-      background: @color-critical;
+      background: var(--search-term-invalid-bg, @search-term-invalid-bg);
     }
 
     .search-errors {
-      color: @color-critical;
+      color: var(--search-term-invalid-bg, @search-term-invalid-color);
       font-weight: bold;
     }
   }
 
   li > select {
-    background: @gray-light;
+    background: var(--search-editor-group-operator-bg, @search-editor-group-operator-bg);
```

Comment:
This isn't for group operators. They are not shown in the editor. It's for logical operators. So use `@search-editor-logical-op-bg` instead please and remove `@search-editor-group-operator-bg`.